.spelling

```diff
@@ -229,6 +229,7 @@ HMN
 HMNLB
 HMN_MTN
 HMN_RVR
+hms-discovery
```

Comment:
Because this is the name of a cronjob, we should put backticks around the places where this is used instead of adding it to the dictionary.

Reply:
Fixed in commit 3c62b31.